### PR TITLE
Adapt AvatarRoom.xml for phones and tablets in portrait and landscape

### PR DESCRIPTION
**Pull request by vasukiarunachalam**
Following changes were made -

1. Changed background.png for better look and feel.
2. Changed AvatarRoomActivity.java to maintain naming convention. ( "newUserButton" to eyeRight")
3. Modified layout/avatar_room.xml to display button in portrait mode.
4. Created layout-land/avatar_room.xml to display in landscape mode.
5. Created values for dimen.xml for different sizes of phones and tablets

For normal devices -
values/dimens.xml
values-land/dimens.xml 

For 5" devices-
values-sw360dp/dimens.xml
values-sw360dp-port/dimens.xml 

For 7" devices-
values-sw600dp/dimens.xml
values-sw600dp-port/dimens.xml 

For 9" and 10" devices-
values-sw720dp/dimens.xml
values-sw720dp-port/dimens.xml

### CODE REVIEW

**Comment by alma-castant** [comment]
This is ready to merge and works well. There are merge conflicts, so please rebase and solve them so I can merge it

**Comment by vasukiarunachalam** [comment]
I have resolved the conflicts and committed the changes. Please review and let me know if it is correct.
Thank you.

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 1]
```diff
@@ -37,81 +39,93 @@
...
-        android:id="@+id/faceRight"
+        android:id="@+id/eyeLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/face"
-        android:layout_below="@id/face"
-        android:layout_marginTop="@dimen/face_right_margin_top"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/eyes"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: Use `@id` here as you are referencing another element. Use `@+id` only when defining a new id

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 2]
```diff
@@ -37,81 +39,93 @@
...
     <ImageButton
-        android:id="@+id/clotheRight"
+        android:id="@+id/eyeRight"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/clothes"
-        android:layout_alignTop="@id/faceRight"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_right_white_18dp"
+        android:layout_below="@+id/eyes"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: Same

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 3]
```diff
@@ -37,81 +39,93 @@
...
     <ImageButton
-        android:id="@+id/hairRight"
+        android:id="@+id/hairLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/face"
-        android:layout_below="@id/hair"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/hair"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: Same

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 4]
```diff
@@ -37,81 +39,93 @@
...
     <ImageButton
-        android:id="@+id/clotheLeft"
+        android:id="@+id/hairRight"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignTop="@id/clotheRight"
-        android:layout_toLeftOf="@id/clotheRight"
-        android:src="@drawable/ic_keyboard_arrow_left_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_right_white_18dp"
+        android:layout_below="@+id/hair"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: Same

File: PowerUp/app/src/main/res/layout/avatar_room.xml [hunk 5]
```diff
@@ -37,81 +39,93 @@
...
     <ImageButton
-        android:id="@+id/hairLeft"
+        android:id="@+id/faceLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignTop="@id/hairRight"
-        android:layout_toLeftOf="@id/hairRight"
-        android:src="@drawable/ic_keyboard_arrow_left_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/face"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout/avatar_room.xml]: Same

**Comment by vasukiarunachalam** [comment]
I made the changes and pushed it, please review.

**Pull request merged**